css/style.css

```diff
@@ -0,0 +1,17 @@
+.container{
+    width:500px;
+    margin:100px auto;
+}
+
+*{
```

Comment:
Body is easier to override than *. * selects every element equally, while body cascades down to the rest of the page.